README.md

```diff
@@ -39,12 +39,12 @@ The following MQTT topics are subscribed to:
 
 | Topic | Description | Values |
 | ----- | ----------- | ------ |
-| am43/&lt;device>/set          | Set the blind position | 'OPEN', 'STOP' or 'CLOSE' |
-| am43/&lt;device>/set_position | Set the blind % position | between 0 and 100. |
+| am43/&lt;device>/set          | Set the blind position (would be deprecated in future release) | 'OPEN', 'STOP' or 'CLOSE' |
+| am43/&lt;device>/set_position | Set the blind position or % position | 'OPEN', 'UP', 'STOP', 'CLOSE', 'DOWN' or a number between 0 and 100. |
 | am43/&lt;device>/status | Get device status on demand (position/light/battery) | Ignored. |
-| am43/enable               | Enable or disable BLE connections | 'off' or 'on'.
+| am43/enable               | Enable = BLE connections AllwaysOn mode, Disable = BLE OnDemand Connections mode | 'off' or 'on'.
```

Comment:
"allways" should be "always".

The rest looks largely ok.

Reply:
Done! :)